.github/workflows/heroku-pull-request.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             RACK_ENV=production \
             LOG_LEVEL=info \
             RAILS_SERVE_STATIC_FILES=enabled \
-            RAILS_MASTER_KEY_NEW=${{ secrets.RAILS_MASTER_KEY_NEW }} \
```

Comment:
where is this RAILS_MASTER_KEY_NEW being referenced in the heroku apps?

Reply:
it's how the rails app in the heroku dyno is decrypting the credentials file. we configure the secret in the [github actions secrets](https://github.com/codeforamerica/vita-min/settings/secrets/actions). the key is injected in this build step into the heroku dyno environment. you can see the value in the 'config vars' section of the [heroku application settings](https://dashboard.heroku.com/apps/gyr-review-app-5256/settings).

Reply:
ah, makes sense!